sections/capoeira/marinheiro_so/index.qmd

```diff
@@ -0,0 +1,122 @@
+---
+title: Marinheiro só
+subtitle: Marin solitaire
+description: Une chanson de capoeira qui conte la vie de marin
+date: 2023-07-10
+date-modified: 07-08-2023
+license: CC BY-SA
+author: 
+  - Z.
+  - PYM
+categories: 
+  - capoeira
+  - chant
+title-block-banner: ???
+image: ???
+image-alt: ???
+---
+
+INTRO
+
+## Le chant
+
+::: {layout-ncol=2}
+
+### Marinheiro só 🇧🇷
+
+| Eu não sou daqui
+| **Marinheiro só**
+| Eu não tenho amor
+| **Marinheiro só**
+| Eu sou da Bahia
+| **Marinheiro só**
+| De São Salvador
+| **Marinheiro só**
+| Mairheiro, marinheiro
+| **Marinheiro só**
+| Quem te ensinou a nadar
+| **Marinheiro só**
+| Foi o tombo do navio
+| **Marinheiro só**
+| Ou foi o balanço do mar
+| **Marinheiro só**
+| La vem, la vem,
+| **Marinheiro só**
+| Como ele vem façeiro
+| **Marinheiro só**
+| Como todo de branco
+| **Marinheiro só**
+| Com seu bonezinho
+
+### Marin solitaire 🇫🇷
+
+| Je ne suis pas d'ici
+| **Marin solitaire**
+| Je n'ai pas d'amour
+| **Marin solitaire**
+| Je suis de Bahia
+| **Marin solitaire**
+| De São Salvador
+| **Marin solitaire**
+| Marin, Marin
+| **Marin solitaire**
+| Qui t'a appris à nager
+| **Marin solitaire**
+| Etait-ce le roulis du navire
+| **Marin solitaire**
+| Ou était-ce la houle de la mer
+| **Marin solitaire**
+| Il arrive, il arrive
+| **Marin solitaire**
+| Comme il est coquet
+| **Marin solitaire**
+| Tout de blanc vêtu
+| **Marin solitaire**
+| Avec sa petite casquette
+| **Marin solitaire**
+
+:::
+
+## Vidéo de référence
+
+{{< https://www.youtube.com/watch?v=mjqG0UlpTb8 >}}
```

Comment:
J'ajouterais avant cette vidéo une petite précision : ici il ne s'agit pas de la version classique de capoeira car c'est accompagné à la guitare (donc c'est pas vraiment la référence à avoir en tête).

La version de Maria Suares est clairement plus fidèle à ce qu'on attend d'un chant de capoeira : https://youtu.be/B2c1E2K0rrQ

Et tu peux ajouter ajouter la version lente et clean avec des explication en anglais : https://youtu.be/wU39vOP2X0U